server/src/test/scala/org/scassandra/server/e2e/CurrentTest.scala

```diff
@@ -1,78 +0,0 @@
-/*
- * Copyright (C) 2016 Christopher Batey and Dogan Narinc
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.scassandra.server.e2e
-
-import com.datastax.driver.core.Cluster
-import com.datastax.driver.core.HostDistance.LOCAL
-import com.datastax.driver.core.exceptions.NoHostAvailableException
-import org.scassandra.server.{ConnectionToServerStub, AbstractIntegrationTest}
-import org.scassandra.server.PrimingHelper._
-
-class CurrentTest extends AbstractIntegrationTest {
```

Comment:
I planned not to as we do so much integration testing via the java client but maybe this one and a few others are valid as we don't test in the it-projects